atflatcontrols/atscrollbar.pas

```diff
@@ -501,12 +593,35 @@ procedure TATScrollbar.Resize;
 end;
 
 
-{$ifdef windows}
+{$ifndef FPC}
 //needed to remove flickering on resize and mouse-over
```

Comment:
but its needed for FPC on WIndows

Reply:
Not sure of the fix needed so I added {$ifdef windows} back in and took {$ifndef FPC} out. If that's wrong let me know.